content/docs/cucumber/varieties.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Cucumber Varieties
+subtitle: "Varieties and Semantic versioning"
+
+weight: 3
+---
+
+Every [Cucumber implementation](/docs/installation) has its own codebase, release cycle and [semver](https://semver.org/) version that is independent from other Cucumber implementations.
+
+The Cucumber team also maintains a list of *varieties* which defines essential characteristics of various Cucumber versions. A Cucumber release may satisfy zero or more characteristics of a variety. If it satisfies all of them it is a member of that variety.
+
+Marking Cucumber releases as part of a variety serves several purposes:
+
+* Define stretch goals for upcoming Cucumber implementations
+* Allow tool vendors to support particular varieties with more ease
+* Documentation becomes easier to read and write as it applies to all releases in the variety
+
+Varieties are named after Wikipedia's [list of Cucumber varieties](https://en.wikipedia.org/wiki/List_of_cucumber_varieties). Higher alphabetical order means more recent.
+
+# Babylon
+
+Characteristics               | [Cucumber-JVM 5.0.0-RC2] | [Cucumber.js master] | [Cucumber-Ruby 4.0.0.rc.2]
```

Comment:
Why is cucumber-js master here instead of a version?

Reply:
For the most recent variety definition (Babylon) my idea is that the Cucumber version in the header should be one of the following:

* A final (non-RC) release, if it satisfies all of the characteristics
* An RC release, if it satisfies one or more charactersitics (with the assumption that the final release will support them all)
* If none of the above, use master

Does that make sense @charlierudolph? If so, I can add that to the description.